### DISCUSSION TITLE
Reads during quorum loss

### DISCUSSION THREAD

**Discussion by anupamdialpad** [etcd-io/etcd#18030] [Q&A] [answered]
Hi,

I am wondering what happens in a 7 member cluster if 4 member goes down? and 1 of the 4 member which went down had the leader in it

I know the writes won't work
Will the reads work?

**Comment by ahrtr** [comment, marked as answer]
> I know the writes won't work
> Will the reads work?

Once a cluster lost the quorum, then it can't serve any write requests and lineariziable read requests. But it can serve serializable read requests, see example below,

```
$ etcdctl get k1 --consistency="s"
```

Please also read https://etcd.io/docs/v3.5/faq/#what-is-failure-tolerance
